libraries/core-react/src/components/TextField/Field.tsx

```diff
@@ -125,13 +122,10 @@ const Adornments = styled.div<AdornmentsType>`
   & div:nth-child(2) {
     margin-left: ${textfield.spacings.left};
   }
-  ${({ multiline, density }) =>
+  margin-top: ${({ theme }) => theme.spacings.top};
```

Comment:
This looks like it might cause a visual bug? `margin-top` was only applied if `multiline` was true...

Reply:
Yes, you are right. Corrected 👍